src/main/java/ch/njol/skript/expressions/base/PropertyExpression.java

```diff
@@ -36,43 +18,58 @@
 /**
  * Represents an expression which represents a property of another one. Remember to set the expression with {@link #setExpr(Expression)} in
  * {@link SyntaxElement#init(Expression[], int, Kleenean, ParseResult) init()}.
- * 
+ *
  * @see SimplePropertyExpression
  * @see #register(Class, Class, String, String)
  */
 public abstract class PropertyExpression<F, T> extends SimpleExpression<T> {
 
+	/**
+	 * A helper method to get the two default property patterns from a property and type parameter.
+	 * i.e. "property of %types%" and "%types%'[s] property"
+	 *
+	 * @param property the name of property
+	 * @param fromType the types that the property should apply to
+	 * @return an array of strings that represent the patterns of the provided property and types
+	 */
+	@SuppressWarnings("ConstantValue")
+	public static String[] getPatterns(String property, String fromType) {
+		if (property == null || fromType == null)
+			return null;
```

Comment:
i would suggest an exception here instead, there's no case in which one of these being null is intended